packages/api/src/useSubscription.ts

```diff
@@ -35,6 +36,13 @@ const useSubscription = <T extends TSocketSubscribableEndpointNames>(name: T) =>
                         setIsLoading(false);
                     }
                 );
+                const subscriber_key = md5(JSON.stringify({ name, payload })).slice(0, 16);
```

Comment:
the funny thing about Map is that it genuinely accepts objects as keys. So no need to have expensive md5 calculations of keys ;) and thats by design

Reply:
Oh, but what's the value then 😅😅😅

Reply:
yep, its fine to do stuff like:
map.set(subscriber, unsubscribe)

and later on:
unsubsribe = map.get(subscriber)

as much as it looks slightly weird - its by design, and internally its executed very fast as every object internally have some sort of identifier (not sure exactly about details of V8 execution, but definitley its fine)

Reply:
Ah got it, thanks 🙌🏻

Reply:
Have updated but `subscribers` type looks a bit overblown now 😁